content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem or cloud data stores and powerful servers,
+which is expensive and time consuming.
+
+![](/img/resource-pool.png) _Fragile and disjointed pool of limited resources_
+
+Minimizing operational cost and complexity can make the difference, for example,
+when:
+
+- Multiple users work on the same shared server.
+- There's a centralized data storage unit or single-entry cluster.
+- There's a single computing environment with access to the data needed to
+  reproduce full [experiments](/doc/user-guide/experiment-management).
+- Distributing GPU time among people or processes for training machine learning
+  models
+
+In DVC, a built-in <abbr>caching</abbr> mechanism already provides individual
```

Comment:
people reading this might not realize I think that cloning DVC project means a Git repo

that they don't deal with some crazy infrastrucutre, etc ... we need to go bit higher I think here and may be in some precious paragraphs

Reply:
OK. I removed details and made other small changes that help clarify for now. But I'll try to go even higher level... ⌛

Reply:
OK I rewrote this part again, PTAL.